src/generic_ui/polymer/contact.html

```diff
@@ -129,7 +129,7 @@
       <uproxy-avatar src='{{ contact.imageData }}' network='{{ contact.network.name }}' showIcon='{{ model.onlineNetworks.length > 1 }}' on-tap='{{ toggle }}'></uproxy-avatar>
       <div flex>
         <div class='name' on-tap='{{ toggle }}'><bdi>{{ contact.name }}</bdi></div>
-        <div class='{{ (isOnline || contact.status!=UserStatus.FRIEND) ? "onlineStatus" : "offlineStatus" }}'
+        <div class='{{ isOnline ? "onlineStatus" : "offlineStatus" }}'
```

Comment:
This was a CSS fix for pending GitHub requests (they were appearing at a lower opacity than expected)